flyway/postgre/migrations/sql/V1_3__Create_lesson2_tables.sql

```diff
@@ -1,69 +1,86 @@
+CREATE TYPE sex_type AS ENUM ('male', 'female');
+
+DROP TABLE IF EXISTS sso_organization;
+DROP TABLE IF EXISTS sso_user;
+DROP TABLE IF EXISTS sso_roles;
+DROP TABLE IF EXISTS sso_user_profile;
+DROP TABLE IF EXISTS sso_organization_user;
+DROP TABLE IF EXISTS sso_user_role;
+
+
 
 CREATE TABLE IF NOT EXISTS sso_organization (
-    organization_id SERIAL PRIMARY KEY NOT NULL,
-    organization_name VARCHAR(255) NOT NULL,
-    status organization_status_type DEFAULT 'active'::organization_status_type NOT NULL,
+    organization_id INT NOT NULL,
+    organization_name VARCHAR(100) NOT NUll,
+    kvp JSON,
+    date_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    date_modified TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    status organization_status_type DEFAULT 'active',
     address VARCHAR(255),
-    contact_number VARCHAR(255),
-    email VARCHAR(255),
+    contact_number VARCHAR(100),
+    email VARCHAR(100),
     country VARCHAR(255),
     state VARCHAR(255),
     city VARCHAR(255),
-    postal_code VARCHAR(255),
-    website_url VARCHAR(255),
-    kvp JSONB,
-    date_created TIMESTAMPTZ DEFAULT now(),
-    date_modified TIMESTAMPTZ DEFAULT now()
+    postal_code VARCHAR(10),
+    organization_sub_name VARCHAR(10),
+    website_url VARCHAR(255)
+    PRIMARY KEY (organization_id)
 );
 
-CREATE TABLE IF NOT EXISTS sso_role (
-    role_id UUID DEFAULT uuid_generate_v4() PRIMARY KEY NOT NULL,
-    role_name VARCHAR(255) NOT NULL,
-    role_description VARCHAR(255),
-    organization_id INT,
-    FOREIGN KEY (organization_id) REFERENCES sso_organization(organization_id)
+CREATE TABLE IF NOT EXISTS sso_user (
+    user_id UUID DEFAULT uuid_generate_v4(),
+    username VARCHAR(100) NOT NULL,
+    password VARCHAR(100) NOT NULL,
+    email VARCHAR(100) NOT NULL,
+    kvp JSON,
+    date_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    date_modified TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    modified_by UUID DEFAULT uuid_generate_v4(),
+    last_logged_in TIMESTAMP,
+    user_settings JSON DEFAULT '{}',
+    status user_status_type DEFAULT 'active'
+    PRIMARY KEY (user_id)
 );
 
-CREATE TABLE IF NOT EXISTS sso_user (
-    user_id UUID DEFAULT uuid_generate_v4() PRIMARY KEY NOT NULL,
-    username VARCHAR(255) NOT NULL,
-    password VARCHAR(255) NOT NULL,
-    email VARCHAR(255) NOT NULL,
-    kvp JSONB,
-    date_created TIMESTAMPTZ DEFAULT now(),
-    date_modified TIMESTAMPTZ DEFAULT now(),
-    modified_by UUID,
-    last_logged_in TIMESTAMPTZ,
-    user_settings JSONB DEFAULT '{}' :: JSONB,
-    status user_status_type DEFAULT 'active' :: user_status_type
+CREATE TABLE IF NOT EXISTS sso_organization_user (
+    organization_id INT NOT NULL,
+    user_id UUID,
+    PRIMARY KEY (organization_id, user_id),
+    FOREIGN KEY (organization_id) REFERENCES sso_organization(organization_id),
+    FOREIGN KEY (user_id) REFERENCES sso_user(user_id)
+);
+
+CREATE TABLE IF NOT EXISTS sso_roles (
+    role_id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
```

Comment:
Ở dưới có primary key rồi nên ở trên ko cần nữa nhé, với thêm NOT NULL cho khóa chính
```suggestion
    role_id UUID DEFAULT uuid_generate_v4() NOT NULL,
```